packages/ui/src/composites/wui-transaction-list-item-loader/styles.ts

```diff
@@ -0,0 +1,14 @@
+import { css } from 'lit'
+
+export default css`
+  :host > wui-flex:first-child {
```

Comment:
I still might need to search for the best approaches to writing CSS with web components. Somehow I don't really feel quite quite good in some cases. 

Please if you see some bad practice in the CSS, let me know.